### PR TITLE
New tooltip for wasta with new model, mako and translations(v2)

### PR DESCRIPTION
**Pull request by alcapat**
Change key name in model and tooltip
Use ressource_exist to test if the image exsit

### CODE REVIEW

**Comment by alcapat** [comment]
jenkins test

**Comment by alcapat** [comment]
Jenkins test

**Comment by AFoletti** [comment]
note that this is needed for today too!

**Comment by procrastinatio** [comment]
Please correct tests and merge.

**Comment by procrastinatio** [comment]
jnekins tests

**Comment by procrastinatio** [comment]
jenkins tests

File: chsdi/models/vector/uvek.py [hunk 1]
```diff
@@ -632,6 +632,37 @@ class MinergieGebaeude(Base, Vector):
...
+    pumpspowerinputmaximum = Column('pumpspowerinputmaximum', Float)
+    enginepowerdemand = Column('enginepowerdemand', Float)
+    fallheight = Column('fallheight', Unicode)
+    the_geom = Column(Geometry2D)
+
+register('ch.bfe.statistik-wasserkraftanlagen', StatistikwasserkraftanlagenNew)
+
+
 class STATISTIKWASSERKRAFTANLAGEN(Base, Vector):
     __tablename__ = 'statistik_wasserkraftanlagen_powerplant'
```
Comment (loicgasser) [inline chsdi/models/vector/uvek.py]: @alcapat you need to remove this model.
If you wanted a smooth update you would have needed to create a new table. it's too late now and as you're defining 2 models pointing to the same table tests fail.

Comment (alcapat) [inline chsdi/models/vector/uvek.py]: @loicgasser i have removed the old model. But tests failed again.

**Comment by alcapat** [comment]
jenkins test

**Comment by procrastinatio** [comment]
Much better

**Pull request merged**